src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
-        <target state="new">Most people who use the Internet have never reflected on what actually happens when one types an address in their web browser and then clicks the &apos;Enter&apos; button. In short, the DNS (Domain Name System) is a database that associates domain names to IP (Internet Protocol) addresses, the same way a phone book associates names to phone numbers. An IP address is a unique series of numbers that identifies every computer that is connected to the Internet. It is similar to the way every telephone has its own number in the telephone network. Such long series of numbers work fine for computers, but for people it is way more difficult to memorize. Thus, the DNS has been developed as an extra layer to map between names that we can remember better and IP addresses used by computers whenever we want to reach a specific website or use other services on the Internet (e.g., mail).</target>
+        <source>Most people who use the Internet have never reflected on what actually happens when one types an address in their web browser and then clicks the 'Enter' button. In short, the DNS (Domain Name System) is a database that associates domain names to IP (Internet Protocol) addresses, the same way a phone book associates names to phone numbers. An IP address is a unique series of numbers that identifies every computer that is connected to the Internet. It is similar to the way every telephone has its own number in the telephone network. Such long series of numbers work fine for computers, but for people it is way more difficult to memorize. Thus, the DNS has been developed as an extra layer to map between names that we can remember better and IP addresses used by computers whenever we want to reach a specific website or use other services on the Internet (e.g., mail).</source>
+        <target state="translated">La plupart des personnes qui utilisent Internet ne se sont jamais demandées ce qui se passe réellement quand elles tapent une adresse dans leur navigateur Web et appuient ensuite sur la touche « entrée ». En résumé le DNS (Domain Name System) est une base de données qui associe des noms de domaine à des adresses IP (Internet Protocol), de la même manière qu’un annuaire associe des noms à des numéros de téléphone. Une adresse IP est un identifiant numérique unique qui identifie chaque ordinateur connecté à Internet, de la même manière que chaque téléphone a son propre numéro dans le réseau téléphonique. Ces identifiants numériques complexes conviennent parfaitement aux ordinateurs, mais sont beaucoup plus difficiles à mémoriser pour des humains. C’est pourquoi le DNS a été développé, pour établir une correspondance entre des noms que nous pouvons plus facilement retenir et les adresses IP utilisées par les ordinateurs lorsqu’on veut accéder à un site Web ou utiliser d’autres services sur Internet (comme le courriel).</target>
```

Comment:
Remove ','
a été développé, -> a été développé

Reply:
I am not sure about this one